Modules/vc-module-pricing/VirtoCommerce.PricingModule.Data/ExportImport2/ExportablePricelist.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+using VirtoCommerce.ExportModule.Core.Model;
+using VirtoCommerce.PricingModule.Core.Model;
+
+namespace VirtoCommerce.PricingModule.Data.ExportImport
+{
+    public class ExportablePricelist : ExportableEntity<ExportablePricelist>
+    {
+        #region Pricelist properties
+        public string Description { get; set; }
+        public string Currency { get; set; }
+        public string OuterId { get; set; }
+        public ICollection<Price> Prices { get; set; }
```

Comment:
These collection properties should not be the part of `ExportablePricelist` object, otherwise these collections would be passed to the client when viewing.
Seems we need to create own object for PricelistFullData, which will be not viewable.
Please remove these properties and PricelistFullData registration for now, full data export will be implemented in VP-255 task.